src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
+    private static final String RESULT = "result";
+    private static final int OPERATION = 0;
+    private static final int VALUE = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
![image](https://user-images.githubusercontent.com/117547096/208415270-7363b765-f0d8-4dab-b2ff-62ebe650f74d.png)

```suggestion
    public void getStatistic(String fromFileName, String toFileName) {
        List<String> dataFromFile = readFromFile(fromFileName);
        String report = createReport(dataFromFile);
        writetoFile(report, toFileName);
    }
```